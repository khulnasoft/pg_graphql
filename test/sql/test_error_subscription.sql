select graphql.resolve(
    query:='subscription Abc { anon }'
)
