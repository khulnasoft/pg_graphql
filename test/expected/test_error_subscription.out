select graphql.resolve(
    query:='subscription Abc { anon }'
)
                           resolve                            
--------------------------------------------------------------
 {"errors": [{"message": "Subscriptions are not supported"}]}
(1 row)

